src/libs/ReportUtils.ts

```diff
@@ -545,6 +546,13 @@ function isReportApproved(report: OnyxEntry<Report>): boolean {
     return report?.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && report?.statusNum === CONST.REPORT.STATUS.APPROVED;
 }
 
+/**
+ * Checks if the supplied report is in Open state and status.
```

Comment:
NAB: slightly more explicit
```suggestion
 * Checks if the supplied report is an expense report in Open state and status.
```